dataprofiler/profilers/profile_builder.py

```diff
@@ -23,6 +23,7 @@
 from .helpers.report_helpers import calculate_quantiles, _prepare_report
 from .profiler_options import ProfilerOptions, StructuredOptions
 
+from tqdm import tqdm
```

Comment:
Import only needs to happen right before use IMO

Reply:
Does this mean I should move this to inside the _update_profile_from_chunk method?

Reply:
Yup

Reply:
Moved it inside